pilot/pkg/serviceregistry/kube/controller/ambient/ambientindex.go

```diff
@@ -125,10 +133,22 @@ type Options struct {
 	LookupNetwork         LookupNetwork
 	LookupNetworkGateways LookupNetworkGateways
 	StatusNotifier        *activenotifier.ActiveNotifier
+	Flags                 FeatureFlags
 
 	Debugger *krt.DebugHandler
 }
 
+// KrtOptions is a small wrapper around KRT options to make it easy to provide a common set of options to all collections
+// without excessive duplication.
+type KrtOptions struct {
```

Comment:
maybe move to the krt pkg, so that all callers can use it

Reply:
krt already provides a functional options pattern, this is a simple syntax helper specific to this package to provide a common set of options all the collections in this package need